src/js/06-code.js

```diff
@@ -86,6 +109,8 @@ document.addEventListener('DOMContentLoaded', function () {
     var language = block.hasAttribute('class') &&
       block.getAttribute('class').match(/language-([a-z0-9-])+/i)[0].replace('language-', '')
 
+    language = casedLang(language)
```

Comment:
I think we should translate language after the if. We are not just updating the casing, some languages are translated (for instance, `csharp` -> `C#`).

Reply:
Ideally, we should only use it in `languageDiv.innerHTML = casedLang(language)`